brave/src/test/java/brave/handler/MutableSpanTest.java

```diff
@@ -91,7 +89,7 @@
     // When exporting into a list, a lambda would usually need to close over the list, which results
     // in a new instance per invocation. Since there's a target type parameter, the lambda for this
     // style of conversion can be constant, reducing overhead.
-    List<Tag> listTarget = new ArrayList<>(span.tagCount());
```

Comment:
in hindsight maybe it is worth putting these back.. they are allocation free ways to get the count..